### PR TITLE
[Components with more than 1 service] LPS-188721 Refactor ModelListener components part II

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-188721

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:search

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:search

File: modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOEntryIdentifiableOSGiService.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.liferay.portal.kernel.module.framework.service.IdentifiableOSGiService;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Joao Victor Alves
+ */
+@Component(service = IdentifiableOSGiService.class)
+public class PLOEntryIdentifiableOSGiService
```
Comment (dantewang) [inline modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOEntryIdentifiableOSGiService.java:25]: @MrJohn1911 This one shares the same issue as mentioned in https://github.com/liferay-core-infra/liferay-portal/pull/4855#pullrequestreview-1506703417

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOEntryIdentifiableOSGiService.java (hunk 1 above)
Comment (MrJohn1911) [reply modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOEntryIdentifiableOSGiService.java:25]: see this [comment](https://github.com/liferay-core-infra/liferay-portal/pull/4855#discussion_r1248233160). Thanks.

**Pull request closed without merging**